.github/workflows/docker-build-push.yaml

```diff
@@ -3,48 +3,62 @@ run-name: ${{ github.actor }} triggered Docker pipeline 🚀
 on:
   push:
     branches:
-    - main
+      - main
```

Comment:
excuse my formatter. As long as we don't have competing formatters, this is fine by me. If we do have competing formatters we can adopt a `.editorconfig`